distributed-press/srv1.distributed.press/roles/install-hypercore/vars/main.yml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
How are these generated?

Reply:
This is the static dat-store user and pass that I put in our password manager and it is encrypted with Ansible vault.